docs/en/infraops/installation.asciidoc

```diff
@@ -1,165 +1,120 @@
+If your metrics data or logs data has non-standard fields, you may need to modify some configuration settings in {kib} to change the default behaviors, such as the index pattern used to query the data, and the timestamp field used for sorting.
+For more information, see {kibana-ref}/infrastructure-ui-settings-kb.html[{infra-ui} UI Settings] and {kibana-ref}/logs-ui-settings-kb.html[{logs-ui} UI Settings].
+// ++ This probably should link to the docs about Configure Source flyout/tab-to-be instead of this, unless there is more to do . . .
 
 [float]
-===== Host Metrics
-
-*CPU Usage*:: Average of `system.cpu.user.pct` added to the average of `system.cpu.system.pct` divided by `system.cpu.cores`
+=== More about container monitoring
 
-*Memory Usage*:: Average of `system.memory.actual.used.pct`
-
-*Load*:: Average of `system.load.5`
-
-*Inbound Traffic*:: Derivative of the max of `system.netowrk.in.bytes` scaled to a 1 second rate
-
-*Outbound Traffic*:: Derivative of the max of `system.netowrk.out.bytes` scaled to a 1 second rate
-
-*Log Rate*:: Derivative of the cumulative sum of the document count scaled to a 1 second rate.
-This metric relies on the same indices as the logs.
-
-
-[float]
-===== Docker Container Metrics
-
-*CPU Usage*:: Average of `docker.cpu.total.pct`
-
-*Memory Usage*:: Average of `docker.memory.usage.pct`
-
-*Inbound Traffic*:: Derivative of the max of `docker.network.in.bytes` scaled to a 1 second rate
-
-*Outbound Traffic*:: Derivative of the max of `docker.network.out.bytes` scaled to a 1 second rate
-
-
-[float]
-===== Kubernetes Pod Metrics
-
-*CPU Usage*:: Average of `kubernetes.pod.cpu.usage.node.pct`
-
-*Memory Usage*:: Average of `kubernetes.pod.memory.usage.node.pct`
-
-*Inbound Traffic*:: Derivative of the max of `kubernetes.pod.network.rx.bytes` scaled to a 1 second rate
-
-*Outbound Traffic*:: Derivative of the max of `kubernetes.pod.network.tx.bytes` scaled to a 1 second rate
+// ++ What does this mean?
 
+If you're monitoring containers, you can use autodiscover to automatically apply configuration changes in response to changes in your containers.
+To learn how, see:
 
+* {filebeat-ref}/configuration-autodiscover.html[{filebeat} autodiscover configuration]
+* {metricbeat-ref}/configuration-autodiscover.html[{metricbeat} autodiscover configuration]
 
 [float]
-==== More about container monitoring
-
-If you're monitoring containers, you can use autodiscover to automatically apply
-configuration changes in response to changes in your containers. To learn how,
-see:
-
-* {filebeat-ref}/configuration-autodiscover.html[{filebeat} autodiscover
-configuration]
-* {metricbeat-ref}/configuration-autodiscover.html[{metricbeat} autodiscover
-configuration]
+=== Running Metricbeat 6.5 or earlier with Kibana 6.6 or later
 
+// ++ Is this needed here since we've just installed the latest version of both Kibana and Metricbeat?
```

Comment:
We may remove this section if it is intended for latest versions, Beats < 6.8 don't work with ES/Kibana 7.X.